.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+<!-- 
+Please make sure you have read the contribution guidelines https://github.com/grafana/k6/blob/master/CONTRIBUTING.md as well as the
+the code of conduct https://github.com/grafana/k6/blob/master/CODE_OF_CONDUCT.md before opening a PR.
+-->
+
+## What?
+
+<!-- A description of the changes this PR brings to the documentation. -->
+
+## Checklist
+
+Please fill in this template:
+- [ ] I have used a meaningful title for the PR.
+- [ ] I have described the changes in made in the "What?" section above.
+- [ ] I have performed a self-review of my changes.
+- [ ] I have run the `make docs` command locally and verified that the changes look good.
+
+Select one of these and delete the others:
+
+If updating the documentation of the most recent release of k6: 
+- [ ] I have made my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
+- [ ] I have reflected my changes in the `docs/sources/next` folder of the documentation.
+- [ ] I have reflected my changes in the folders of the previous k6 versions of the documentation.
```

Comment:
```suggestion
- [ ] I have made my changes in the `docs/sources/next` folder of the documentation.
- [ ] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
```

@oleiade I don't know if we discussed this last week, but I think this is something we can decide on about how many previous versions we want to support. I would say for this use case, maybe we can just point users to make changes to `next` and the `v{most_recent_release}` folder. What do you think?

Reply:
I see what you mean. If I remember correctly, in this context, I meant to cater to the specific case where you might be fixing something incorrect in the docs and "backporting" the fix to previous versions that might have been affected. 

How might we improve the phrasing to reflect that?
Also: I'd be fine with dropping it all in all if you prefer for us not to mention it at all 👍🏻

Reply:
I have rephrased the element accordingly 👍🏻 
Let me know if you think we should still remove it 🙇🏻

Reply:
Thanks @oleiade! And sorry for the delay here, I like how you rephrased it. 😄